cmd/metal-api/internal/service/machine-service.go

```diff
@@ -1826,7 +1830,7 @@ func MachineLiveliness(ds *datastore.RethinkStore, logger *zap.SugaredLogger) er
 		case metal.MachineLivelinessDead:
 			dead++
 			p.Dead++
-		default:
+		case metal.MachineLivelinessUnknown:
```

Comment:
Is this the only case we count as unknown ?

Reply:
Yep